### PR TITLE
Additional replaces for Snapable --> Snappable typo fix

### PR DESCRIPTION
**Pull request by ropg**

### CODE REVIEW

File: src/lv_api_map.h [hunk 1]
```diff
@@ -22,7 +22,7 @@ extern "C" {
 #define LV_NO_TASK_READY        LV_NO_TIMER_READY
 #define LV_INDEV_STATE_REL      LV_INDEV_STATE_RELEASED
 #define LV_INDEV_STATE_PR       LV_INDEV_STATE_PRESSED
-#define LV_OBJ_FLAG_SNAPABLE    LV_OBJ_FLAG_SNAPPABLE   /*Fixed typo*/
+#define LV_OBJ_FLAG_SNAPPABLE    LV_OBJ_FLAG_SNAPPABLE   /*Fixed typo*/
```
Comment (kisvegabor) [inline src/lv_api_map.h]: It shouldn't be required.

**Review by kisvegabor — COMMENTED** [review]
I should have used search and replace instead of refactoring...

Thank you!

File: src/lv_api_map.h (hunk 1 above)
Comment (ropg) [reply src/lv_api_map.h]: Yeah, I was just fixing that one. :)

**Comment by kisvegabor** [comment]
Merging! :slightly_smiling_face:

**Pull request merged**